tests/TESTLIST

```diff
@@ -310,7 +310,7 @@ dhcp-rfc3004	dhcp-rfc3004.pcap	dhcp-rfc3004-v.out	-v
 dhcp-rfc5859	dhcp-rfc5859.pcap	dhcp-rfc5859-v.out	-v
 dhcp-mud	dhcp-mud.pcap		dhcp-mud.out	-vv
 
-# MEDSA tests
+# MEDSA (NTP?) tests
```

Comment:
This is a sample of Marvell EDSA link type decoding, it just happens to contain IP/UDP/NTP among other packets.